### PR TITLE
Protect admin routes

### PR DESCRIPTION
**Pull request by grace-fillmore**

### CODE REVIEW

File: src/features/Account/EditAccountForm.tsx
Comment (samohansen) [inline src/features/Account/EditAccountForm.tsx:1]: Was this replaced somewhere else?

File: src/features/Account/EditProfileInterface.tsx
Comment (samohansen) [inline src/features/Account/EditProfileInterface.tsx:1]: I guess these next files answer my previous question :)

**Review by samohansen — APPROVED** [review]
I didn't get to go through all of this but I am going to approve it given that it deploys successfully and that Grace is the best developer by a mile. :)

I have to run but can come back to review code later (if desired).

File: src/features/Account/EditAccountForm.tsx
Comment (grace-fillmore) [reply src/features/Account/EditAccountForm.tsx:1]: yes

File: src/features/Account/EditProfileInterface.tsx
Comment (grace-fillmore) [reply src/features/Account/EditProfileInterface.tsx:1]: oh oops haha

**Pull request merged**